apps/barista-design-system/src/pages/single-page/components/toc/toc.html

```diff
@@ -4,42 +4,44 @@
     (click)="_expandTocMenu()"
     [ngClass]="{ 'ba-is-active': _expandToc }"
   >
-    <a
-      href="#"
+    <p
       class="ba-toc-header ba-expandable-trigger ba-expandable-trigger-right"
       data-initialized="true"
-      >On this page</a
     >
+      On this page
+    </p>
     <div class="ba-toc-content ba-expandable-content">
       <ul class="ba-toc-list ba-list">
         <li
-          *ngFor="let heading of _headings$ | async"
+          *ngFor="let headline of tocitems.headlines"
           class="ba-toc-item"
-          [class.ba-toc-item-active]="
-            _activeItems && _activeItems[0] === heading
-          "
+          [class.ba-toc-item-active]="_activeItems.headlines[0] === headline"
         >
           <a
             #headline
-            href="{{ heading.href }}"
+            id="{{ headline.id }}"
+            href="#{{ headline.id }}"
```

Comment:
this does not work – you would link to your self and not to the headline in the content if you put there the id.